### PR TITLE
Create feature: add new reservation - Student C

### PR DESCRIPTION
**Pull request by david-lafontant**
Implement the "add new reservation" form on the Reservations pop up according to the wireframe:
wireframe

### CODE REVIEW

**Comment by david-lafontant** [comment]
Guys, it would be nice, if you could merge this PR if you approved it.

**Comment by denscholar** [comment]
@david-lafontant Please kindly resolve conflicts. this branch has conflicts that need to be resolved before it can be merged with the development branch. Thank you.

**Review by denscholar — CHANGES_REQUESTED** [review]
@david-lafontant You have done a great job. However, it would be nice if you consider the following optional changes as commented. ♻️

**Review by Aduda-Boaz — COMMENTED** [review]
I also have an opinion that we should leave these as default. We will be flagged.

File: .eslintrc.json
Comment (david-lafontant) [reply .eslintrc.json:26]: Asked change implemented

**Review by denscholar — APPROVED** [review]

**Comment by denscholar** [comment]
Thank you for resolving the conflict. I have nothing else other than to merge it to the development branch.

**Pull request merged**
